example/02-kubeconfig.yaml

```diff
@@ -0,0 +1,30 @@
+# SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
```

Comment:
The current naming scheme makes it think that I would have to apply _both_ YAML files. But I guess they are _alternatives_, so maybe `01-kubeconfig.yaml` and `01-kubeconfig-legacy.yaml` or something along the lines might be better?